src/app/tests/suites/certification/Test_TC_S_3_1.yaml

```diff
@@ -29,141 +29,275 @@ tests:
+          [1706722808.568356][5404:5404] CHIP:DMG:                                 CommandId = 0x0,
+          [1706722808.568366][5404:5404] CHIP:DMG:                         },
+          [1706722808.568381][5404:5404] CHIP:DMG:
+          [1706722808.568391][5404:5404] CHIP:DMG:                         CommandFields =
+          [1706722808.568403][5404:5404] CHIP:DMG:                         {
+          [1706722808.568414][5404:5404] CHIP:DMG:                                 0x0 =
+          [1706722808.568426][5404:5404] CHIP:DMG:                                 {
+          [1706722808.568439][5404:5404] CHIP:DMG:                                         0x0 = 417,
+          [1706722808.568453][5404:5404] CHIP:DMG:                                         0x1 = 0,
+          [1706722808.568465][5404:5404] CHIP:DMG:                                         0x2 = [
+          [1706722808.568482][5404:5404] CHIP:DMG:                                                         0xa0, 0xa1, 0xa2, 0xa3, 0xa4, 0xa5, 0xa6, 0xa7, 0xa8, 0xa9, 0xaa, 0xab, 0xac, 0xad, 0xae, 0xaf,
+          [1706722808.568498][5404:5404] CHIP:DMG:                                         ] (16 bytes)
+          [1706722808.568511][5404:5404] CHIP:DMG:                                         0x3 = 1110000,
+          [1706722808.568523][5404:5404] CHIP:DMG:                                         0x4 = [
+          [1706722808.568539][5404:5404] CHIP:DMG:                                                         0xb0, 0xb1, 0xb2, 0xb3, 0xb4, 0xb5, 0xb6, 0xb7, 0xb8, 0xb9, 0xba, 0xbb, 0xbc, 0xbd, 0xbe, 0xbf,
+          [1706722808.568553][5404:5404] CHIP:DMG:                                         ] (16 bytes)
+          [1706722808.568578][5404:5404] CHIP:DMG:                                         0x5 = 1110001,
+          [1706722808.568594][5404:5404] CHIP:DMG:                                         0x6 = [
+          [1706722808.568610][5404:5404] CHIP:DMG:                                                         0xc2, 0xc1, 0xc2, 0xc3, 0cd4, 0xc5, 0xc6, 0xc7, 0xc8, 0xc9, 0xca, 0xcb, 0xcc, 0xcd, 0xce, 0xcf,
+          [1706722808.568623][5404:5404] CHIP:DMG:                                         ] (16 bytes)
+          [1706722808.568633][5404:5404] CHIP:DMG:                                         0x7 = 1110002,
+          [1706722808.568645][5404:5404] CHIP:DMG:                                 },
+          [1706722808.568661][5404:5404] CHIP:DMG:                         },
+          [1706722808.568671][5404:5404] CHIP:DMG:                 },
+          [1706722808.568695][5404:5404] CHIP:DMG:
+          [1706722808.568704][5404:5404] CHIP:DMG:         ],
+          [1706722808.568729][5404:5404] CHIP:DMG:
+          [1706722808.568739][5404:5404] CHIP:DMG:         InteractionModelRevision = 11
+          [1706722808.568747][5404:5404] CHIP:DMG: },
+
+
+          Before adding the Groups 0x0001 execute below command
```

Comment:
I see in the testplan spec it outlines different steps here https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/cluster/scenes.adoc#pre-conditions.

Where this seems to be on group together command I think it would be easier for QA to follow if this was broke into sub steps with their own label to make it easier to follow

Reply:
Broke the steps into 5 preconditioning steps.